Documentation/network/servicemesh/gateway-api/host-network-mode.rst

```diff
@@ -0,0 +1,43 @@
+Host network mode
+#################
+.. note::
+    Supported since Cilium 1.16+
+
+This mode allows you to expose the Cilium GatewayAPI Gateway
+directly on the host network.
+This is useful in some edge cases where one don't want or can't to use
+LoadBalancer type services (dev environments, cluster-external
+loadbalancer, ...).
+
+The host network mode can be enabled via Helm flag
+``gatewayAPI.hostNetwork.enabled=true``).
+Once enabled, the host network ports can be specified on the ``Gateway``
+resources via ``spec.listeners.port``. The port must be unique per ``Gateway``
+resource and higher than ``1023``.
+
+.. warning::
+    * The listener is exposed on all interfaces (``0.0.0.0`` for IPv4 and/or ``::`` for IPv6).
+    * Be aware that misconfiguration might result in port clashes. Configure unique ports which are still available.
+    * It's mandatory to use ports higher than ``1023`` as the Cilium L7 proxy process `does not have any Linux capabilities`_ (so no ``CAP_NET_BIND_SERVICE``) and is therefore not allowed to listen on privileged ports.
+
+.. _does not have any Linux capabilities: https://github.com/cilium/proxy/pull/315
```

Comment:
FYI: this PR alone doesn't solve the issue. in addition we need to 

- add the capability to the containers (proxy (daemonset mode) and agent (embedded mode))
- pass the new flag to the envoy starter

Reply:
Right, that is a valid point. I can also link to the other to-be-created PRs once they exist.

Reply:
PRs are open
- changing default port for shared and dedicated ingress from `80` to unprivileged `8080`: https://github.com/cilium/cilium/pull/32159
- Support binding to privileged ports: https://github.com/cilium/cilium/pull/32158

Thanks Philip!